Files/Views/PaneHolderPage.xaml.cs

```diff
@@ -36,6 +38,8 @@ public sealed partial class PaneHolderPage : Page, IPaneHolder, ITabItemContent,
 
         private readonly GridLength CompactSidebarWidth;
 
+        public ICommand EmptyRecycleBinCommand { get; private set; }
```

Comment:
Shouldn't there should be a reference to `BaseLayoutCommandsViewModel`?

Reply:
Hmm.. That command appeared after resolving merge issues 🤔

Reply:
It's something we can look at later on, it shouldn't hold us back from merging this PR.